packages/log/src/Logger.ts

```diff
@@ -15,7 +15,9 @@ class Logger {
     this.debug = silent;
     this.debug2 = silent;
 
-    this.setLogLevel(level);
+    if (level !== undefined) {
```

Comment:
I don't think you should allow `level` to be `undefined` at all, just allow a `number` in the constructor.

Really the `name` should be optional, and should be the second parameter, then level is optional and defaults to the defaultLogLevel()